src/main/java/seedu/tr4cker/logic/commands/CountdownCommand.java

```diff
@@ -46,6 +47,14 @@
             + "Examples: \n"
             + COMMAND_WORD + " " + PREFIX_COUNTDOWN_DELETE + "1\n";
 
+    public static final String MESSAGE_COUNT_DAYS_USAGE = COMMAND_WORD
+            + ": Count the number of events in the specified number of days\n"
+            + "Parameters: "
+            + PREFIX_COUNTDOWN_DAYS + "DAYS\n"
+            + "Note: DAYS must be a valid non-negative number!\n"
```

Comment:
Not sure if you should accept 0 as a valid argument because Users might be confused whether to use 0 or 1 to look at the events that are in <24 hours.

Reply:
would the use of a  unique "today" parameter be better?